BlazarUI/app/scripts/stores/buildsStore.js

```diff
@@ -1,5 +1,29 @@
 import Reflux from 'reflux';
 import BuildsActions from '../actions/buildsActions';
+import {pluck} from 'underscore';
+
+
+function getBuildsIds(builds) {
+  return pluck(pluck(builds, 'module'), 'id');
+}
+
+function updateBuilds(latest, builds) {
+
+  const latestIds = getBuildsIds(latest);
+
+  for (let i = 0, len = builds.length; i < len; i++) {
+    for (let g = 0, len = latestIds.length; g < len; g++) {
+      if (latestIds[g] === builds[i].module.id) {
```

Comment:
Only modules already in builds are updated, so if a new module is created it won't be added to `builds` this way, but it would when fetching the complete list.